README.md

```diff
@@ -424,113 +264,134 @@ $ lukso validator deposit \
+
+### `status`
+
+Shows the LUKSO node client processes that are currently running.
+
+### `reset`
+
+| Flag      | Description                 |
+| --------- | --------------------------- |
+| --mainnet | Resets LUKSO's mainnet data |
+| --testnet | Resets LUKSO's testnet data |
+| --devnet  | Resets LUKSO's devnet data  |
+
+### `validator init`
+
+Initializes the validator with keys.
+
+| Flag                             | Description                                 |
+| -------------------------------- | ------------------------------------------- |
+| --max-txs-per-block              | Maximum amount of txs sent per single block |
+| --validator-wallet-dir           | Location of a generated wallet              |
+| --validator-keys-dir             | Path to your validator keys                 |
+| --validator-wallet-password-file | Path to your password file                  |
+
+### `validator deposit`
+
+Makes a deposit to the deposit bridge contract.
+
+| Flag                             | Description                                    |
+| -------------------------------- | ---------------------------------------------- |
+| --deposit-data-json [string]     | Defines the path to the deposit JSON file      |
+| --gas-price [string]             | Defines the gas price in integers as string    |
+| --rpc [string]                   | Defines the RPC URL on deposit                 |
+| --genesis                        | Executes deposit to genesis validator contract |
+| --start-from-index [int]         | Start deposit from specific block index        |
+| --max-txs-per-block              | Maximum amount of txs sent per single block    |
+| --validator-wallet-dir           | Location of a generated wallet                 |
+| --validator-keys-dir             | Path to your validator keys                    |
+| --validator-wallet-password-file | Path to your password file                     |
 
-### PR Testing
+## Available Global Options
 
-This repository has a CI/CD pipeline set up that will automatically build a script to install a new version of the LUKSO CLI globally.
+Global options can be added behind a command to allow different modifications to its execution.
 
-- Should only be used for testing
-- Will overwrite the LUKSO CLI that is currently installed
+| Global Option | Description                                        |
+| ------------- | -------------------------------------------------- |
+| --help, -h    | Show help page of the command provided in the call |
 
-> For PR builds please use the separate PR's GH deployment
+## Available Flags
 
-#### Using the LUKSO CLI URL
+Flags can be added behind a command to allow further command specifications.
 
-```sh
-# Might need admin access by typing `sudo` in front of the command
-curl https://install.lukso.network/36 | sh
-```
+> Global Flags containting [string] are awaiting a string input in quotes.
+> Global Flags containting [int] are awaiting a string input without quotes.
```

Comment:
This has to go above the flag tables or within the specific command heading using [string] and [int]

Reply:
thks - done